src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/QuestionSteps.java

```diff
@@ -461,4 +463,11 @@ private String getQuestionRoundEndpoint(OnlineHearing onlineHearing, Integer rou
 
         return baseUrl + url;
     }
+
+    @Given("^the question expiry date has expired$")
```

Comment:
Basically, just cheat and set the expiry to be now, so that the trigger will fire